front/src/assets/styles/scss/mixins/_buttons.scss

```diff
@@ -49,14 +49,14 @@
     }
     border-color: $active-border;
 
-    &:focus {
-      // Avoid using mixin so we can pass custom focus shadow properly
-      @if $enable-shadows and $btn-active-box-shadow != none {
-        box-shadow: $btn-active-box-shadow, 0 0 0 $btn-focus-width rgba(mix(color-yiq($background), $border, 15%), .5);
-      } @else {
-        box-shadow: 0 0 0 $btn-focus-width rgba(mix(color-yiq($background), $border, 15%), .5);
-      }
-    }
+    //&:focus {
+    //  // Avoid using mixin so we can pass custom focus shadow properly
+    //  @if $enable-shadows and $btn-active-box-shadow != none {
+    //    box-shadow: $btn-active-box-shadow, 0 0 0 $btn-focus-width rgba(mix(color-yiq($background), $border, 15%), .5);
+    //  } @else {
+    //    box-shadow: 0 0 0 $btn-focus-width rgba(mix(color-yiq($background), $border, 15%), .5);
+    //  }
+    //}
```

Comment:
Remove?

Reply:
it's styles core, maybe we will need it later

Reply:
Then can you add a note about why it's commented out, so someone who finds this later will know?

Reply:
ok